src/elements/timetable-occupancy/timetable-occupancy.visual.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { html } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../core/testing/private.js';
+
+import './timetable-occupancy.js';
+
+describe(`sbb-timetable-occupancy`, () => {
+  const cases = {
+    firstClassOccupancy: ['high', 'low'],
+    secondClassOccupancy: ['medium', 'none'],
```

Comment:
Could we also check for undefined case? if there is only one class this happens
```suggestion
    firstClassOccupancy: ['high', 'low', undefined],
    secondClassOccupancy: ['medium', 'none', undefined],
```